test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -88,4 +88,21 @@
     result = run_cmd(@cmd + params)
     assert_equal(result.exit_code, 0)
   end
+
+  it "performs incremental update with packages" do
+    params = ['--content-view-version-id=5', '--packages=bla', '--lifecycle-environment-ids=1']
+
+    expect_generic_search(
+      :packages, params: {search: "name = \"bla\""}, returns: {'id' => 5}
+    ).at_least_once
```

Comment:
`at_least_once` allows for multiple searches. It should work without it (thus expecting only one call).